packages/hardhat-core/src/internal/hardhat-network/provider/node.ts

```diff
@@ -1358,14 +1358,18 @@ If you are using a wallet or dapp, try resetting your wallet's accounts.`
    */
```

Comment:
Unrelated to this PR, but is this comment up to date?

Reply:
Good catch. Deleted it.